go-controller/pkg/informer/informer_test.go

```diff
@@ -130,6 +130,13 @@ var _ = Describe("Informer Event Handler Tests", func() {
 		Expect(err).NotTo(HaveOccurred())
 
 		Consistently(func() int32 { return atomic.LoadInt32(&deletes) }).Should(Equal(int32(0)), "deletes")
+		Eventually(func() (bool, error) {
```

Comment:
Could you explain why this fixes the problem? It seems to me like this should instantly (or nearly) evaluate to true and proceed to L140 as it did before. If the goal is to wait 2 seconds, then we could just add that to the `Eventually` condition on L140. 

Also, I would assume the bug is somehow linked to our informers (I suspect this is why @dave-tucker created https://github.com/ovn-org/ovn-kubernetes/pull/1824 for debugging it), meaning the addFunc does sometimes not get called. I thus don't see how waiting for the pod to exist in the API server would change that.

Reply:
FWIW, I've added more info on #1824 that explains what I think is happening... I don't think it's the code that's wrong, but the machine it's running on is too busy.

Reply:
Understood, thanks for the explanation. I guess we merge #1824 and close this in that case.

Reply:
thanks, i was just trying to add some more time and thought it would take some extra time to see the pod there, @dave-tucker is obviously better. closing this